packages/env/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Restrict `WP_TESTS_DOMAIN` constant to just hostname and an entire URL (with scheme, port, and path).
```

Comment:
```suggestion
- Restrict `WP_TESTS_DOMAIN` constant to just hostname rather than an entire URL (e.g. it now excludes scheme, port, etc.)
```

noticed a typo, and made a separate clarification if that makes sense to you :)

Reply:
Oops. Thank you.